guides/common/modules/snip_red-hat-helper-apps.adoc

```diff
@@ -0,0 +1,11 @@
+ifdef::upgrading-connected,upgrading-disconnected[]
+For interactive upgrade instructions, you can use the {ProjectName} Upgrade Helper on the Red{nbsp}Hat Customer Portal.
```

Comment:
Is "interactive upgrade instructions" meant to be "interactive instructions for performing the upgrade" or "instructions for performing an interactive upgrade"? Either way, it would be less ambiguous to spell it out, I think.

Reply:
The first option is the one we want, updated.